docker-compose-earthly.yml

```diff
@@ -41,6 +41,7 @@ services:
       - KUBERPULT_GIT_BRANCH=main
       - KUBERPULT_API_ENABLE_DESPITE_NO_AUTH=false
       - KUBERPULT_BATCH_CLIENT_TIMEOUT=2m
+      - KUBERPULT_DEX_RBAC_POLICY_PATH=/etc/policy.csv
```

Comment:
this is not mounted to the frontend in the kuberpult chart yet, just so we do not forget!